plugins/circleci/src/circleci-config.ts

```diff
@@ -106,7 +106,11 @@ export default abstract class CircleCiConfigHook extends Hook<CircleCIState> {
 
     state.jobs[this.job] = this.jobOptions
     if (this.addToNightly) {
-      state.nightlyJobs[this.job] = this.jobOptions
+      const jobOptionsWithoutFilters = {
+        ...this.jobOptions,
+        filters: undefined
+      }
+      state.nightlyJobs[this.job] = { ...jobOptionsWithoutFilters }
```

Comment:
**question**: why do we clone `jobOptionsWithoutFilters` (by spreading it) again here?

Reply:
Good question! I got in the flow of spreading. I will remove it now 👍